letsencrypt/cli.py

```diff
@@ -1506,6 +1705,8 @@ def main(cli_args=sys.argv[1:]):
         displayer = display_util.NoninteractiveDisplay(sys.stdout)
     elif config.text_mode:
         displayer = display_util.FileDisplay(sys.stdout)
+    elif config.verb == "renew":
```

Comment:
We should set `config.namespace.noninteractive_mode = True` here so things like https://github.com/letsencrypt/letsencrypt/pull/2338/files#diff-9c91e384e95127e1ff7a8b0e8bc9d521R94 work correctly.